server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -3948,7 +3948,7 @@ public UsageRecordResponse createUsageResponse(Usage usageRecord, Map<String, Se
                 final StringBuilder builder = new StringBuilder();
                 builder.append("VMSnapshot usage");
                 if (vmSnapshotVO != null) {
-                    builder.append(" Id: ").append(vmSnapshotVO.getId()).append(" (").append(vmSnapshotVO.getUuid()).append(") ");
+                    builder.append(" Id: ").append(vmSnapshotVO.getUuid());
```

Comment:
@DaanHoogland 
`.append(") ");` ?

Reply:
what do you mean @weizhouapache ?

Reply:
@DaanHoogland 
line 3994 has `append(") ");` at the end, should it be removed ?

Reply:
ah good catch, thanks